client/src/components/NavBar/NotificationsMenu/NotificationsMenuItems/NotificationsMenuItems.tsx

```diff
@@ -0,0 +1,54 @@
+import useStyles from '../useStyles';
+import { Typography, MenuItem, Paper, ListItemText, ListItemAvatar, Avatar, Box } from '@material-ui/core';
+import { Notification } from '../../../../interface/Notification';
+
+interface Props {
+  notifications: [Notification];
+}
+
+export default function NotificationsMenuItems({ notifications }: Props): JSX.Element {
+  const classes = useStyles();
+  const defaultLogoURL = 'https://res.cloudinary.com/dalisapxa/image/upload/v1630880229/DEV/logo_nywmrf.png';
+
+  return (
+    <Box>
+      <Paper>
+        {notifications &&
+          notifications.map((notification, index) => (
+            <Box key={index} paddingLeft={'4px'}>
```

Comment:
Given we can receive notifications and so indexes change this is actually an anti-pattern to use indexes as keys - please use something more unique such as the notification id.